cmd/finch/virtual_machine_status.go

```diff
@@ -0,0 +1,61 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/runfinch/finch/pkg/command"
+	"github.com/runfinch/finch/pkg/flog"
+	"github.com/runfinch/finch/pkg/lima"
+
+	"github.com/spf13/cobra"
+)
+
+func newStatusVMCommand(limaCmdCreator command.LimaCmdCreator, logger flog.Logger) *cobra.Command {
+	statusVMCommand := &cobra.Command{
+		Use:   "status",
+		Short: "Status of the virtual machine",
+		RunE:  newStatusVMAction(limaCmdCreator, logger).runAdapter,
+	}
+
+	return statusVMCommand
+}
+
+type statusVMAction struct {
+	creator command.LimaCmdCreator
+	logger  flog.Logger
+}
+
+func newStatusVMAction(creator command.LimaCmdCreator, logger flog.Logger) *statusVMAction {
+	return &statusVMAction{creator: creator, logger: logger}
+}
+
+func (sva *statusVMAction) runAdapter(cmd *cobra.Command, args []string) error {
+	return sva.run()
+}
+
+func (sva *statusVMAction) run() error {
+	status, err := lima.GetVMStatus(sva.creator, sva.logger, limaInstanceName)
+	if err != nil {
+		return err
+	}
+	switch status {
+	case lima.Running:
+		fmt.Println("Running")
+		return nil
+	case lima.Nonexistent:
+		fmt.Println("Notexistent")
+		return nil
+	case lima.Stopped:
+		fmt.Println("Stopped")
+		return nil
+	case lima.Unknown:
+		fmt.Println("Unknown")
+		return nil
```

Comment:
If the err returned by `GetVMStatus` is nil, but `GetVMStatus` also returns an `Unknown` status with it, then probably there's a bug. As a result, I think we can return an error for this case by incorporating the suggested changes in https://github.com/runfinch/finch/pull/83#discussion_r1038434102. WDYT?

```suggestion
```

Reply:
That's a good point to know, that there is a bug when `Unknown` is returned. I added an error return for this scenario.